src/backend/utils/storage.js

```diff
@@ -69,13 +72,26 @@ module.exports = {
    */
   removeFeed: async (id) => {
     const key = createFeedKey(id);
+    // If feed is in t:feeds
+    if (await redis.sismember(feedsKey, id)) {
+      await redis
+        .multi()
+        .hdel(key, 'id', 'author', 'url', 'user', 'link', 'etag', 'lastModified')
+        .srem(feedsKey, id)
+        .exec();
+    }
+    // If feed is in t:feeds:flagged
```

Comment:
Should we wrap this in an `else` since it's only in one or the other, not both, right?